doc/manual/metaprogramming.rst

```diff
@@ -45,19 +45,29 @@ each string into an object called an expression, represented by the Julia type
 
     julia> typeof(ex1)
     Expr
+    
 
 :obj:`Expr` objects contain three parts:
 
-- a ``Symbol`` identifying the kind of expression. A symbol is an
-  `interned string <https://en.wikipedia.org/wiki/String_interning>`_
+.. doctest::
+
+    julia> fieldnames(Expr)
+    3-element Array{Symbol,1}:
+     :head
+     :args
+     :typ 
+
+- `head` -- a ``Symbol`` identifying the 
+  `kind of expression <http://docs.julialang.org/en/release-0.5/devdocs/ast/#expr-types>`_. 
+  A symbol is an `interned string <https://en.wikipedia.org/wiki/String_interning>`_
```

Comment:
I disagree with the characterization of a Symbol as an "interned string":

http://stackoverflow.com/questions/23480722/what-is-a-symbol-in-julia/23482257#23482257

Maybe use this line instead: "A symbol is used to represent a variable in metaprogramming". I realize you're just moving text around, but I guess we can improve it while we're at it.